package.json

```diff
@@ -1,9 +1,17 @@
 {
   "name": "modv",
   "productName": "modV",
-  "author": "vcync",
+  "author": "Sam Wray",
```

Comment:
Why not leave it as `vcync`?

Reply:
I used a generator for the basic template. Didn't realise it had changed this - I'll update!